### PR TITLE
added support to initialize the datepicker from a string

### PR DESCRIPTION
**Pull request by daldegam**

### CODE REVIEW

File: spec/directives/dateInput.directive.spec.js [hunk 1]
```diff
@@ -86,4 +86,22 @@ describe('directive: pa-date-input', function() {
 
   });
 
+  describe('test initialize from string', function() {
```
Comment (denislins) [inline spec/directives/dateInput.directive.spec.js]: We should choose a better title for this describe block. It does not clearly states what the block tests. Maybe something like 'when the model is a string'.

File: src/scripts/controllers/container.controller.js [hunk 2]
```diff
@@ -56,6 +56,9 @@
             this.ngModel = {};
           } else if (this.ngModel instanceof Date) {
             this.ngModel.setHours(0, 0, 0, 0);
+          } else if (typeof(this.ngModel) === 'string' || this.ngModel instanceof String) {
```
Comment (denislins) [inline src/scripts/controllers/container.controller.js:67]: I think that these two conditions test the same situation. A simple `else if (typeof this.ngModel === 'string')` would suffice.

**Comment by denislins** [comment]
Awesome PR! We can merge it once we deal with the observations.

File: spec/directives/dateInput.directive.spec.js (hunk 1 above)
Comment (daldegam) [inline spec/directives/dateInput.directive.spec.js]: I will change this

File: src/scripts/controllers/container.controller.js (hunk 2 above)
Comment (daldegam) [inline src/scripts/controllers/container.controller.js:67]: Not exactly, there are specific cases that would require these two comparisons. See the example:
image

File: spec/directives/dateInput.directive.spec.js (hunk 1 above)
Comment (daldegam) [inline spec/directives/dateInput.directive.spec.js]: Done :)

**Pull request merged**